src/jitlayers.cpp

```diff
@@ -99,7 +99,7 @@ static void addOptimizationPasses(T *PM)
     PM->add(createInstructionCombiningPass()); // Clean up after the unroller
 #endif
     PM->add(createGVNPass());                  // Remove redundancies
-    //PM->add(createMemCpyOptPass());            // Remove memcpy / form memset
+    PM->add(createMemCpyOptPass());            // Remove memcpy / form memset
```

Comment:
https://github.com/JuliaLang/julia/commit/123a0e1420b0274c36be690e318b5495d8fbab79 was quite some time ago. Was it more expensive in terms of compile time then vs now?